src/controllers/ownerMessageController.js

```diff
@@ -1,3 +1,5 @@
+const hasPermission = require("../utilities/permissions");
```

Comment:
```suggestion
const { hasPermission } = require("../utilities/permissions");
```
Incorrect import function `hasPermission`